benchmarks/dynamo/common.py

```diff
@@ -1522,6 +1527,14 @@ def forward(self, *args):
 
             model = WrapperModel(model, list(example_inputs.keys()))
 
+        if self.use_experimental_patch:
+            import torch_onnx
+
+            torch_onnx.patch_torch(error_report=True, profile=True)
+        elif "torch_onnx" in sys.modules:
+            # make sure the patch is not in effect
+            torch_onnx.unpatch_torch()
```

Comment:
Is torch_onnx in namespace here? I assume so with sys.modules? Just double checking

Reply:
It looks working locally, but we can also importing it anyway, but patching/unpatching with if/else?

Reply:
We can maybe do

```
try:
    import torch_onnx
    torch_onnx.unpatch_torch()
except ImportError:
    pass
```